ads_plugin.py

```diff
@@ -0,0 +1,190 @@
+import wx
+import wx.lib.agw.hyperlink as hl
+
+import fsleyes.controls.controlpanel as ctrlpanel
+import fsleyes.actions.loadoverlay as ovLoad
+
+import numpy as np
+import nibabel as nib
+from PIL import Image
+import os
+import json
+
+import AxonDeepSeg
+from AxonDeepSeg.apply_model import axon_segmentation
+
+class Ads_control_v2(ctrlpanel.ControlPanel):
```

Comment:
don't call it "_v2"
also, class don't use underscore, name should be something like AdsControl